EasyCommands/BlockHandlers/BlockHandlers.cs

```diff
@@ -71,9 +76,34 @@ public static class BlockHandlerRegistry {
                 return blockHandlers[blockType];
             }
 
-            public static List<Object> GetBlocks(Block blockType, Func<IMyTerminalBlock, bool> selector) => blockHandlers[blockType].GetBlocks(selector);
+            public static void invalidateCaches() {
+                PROGRAM.invalidBlockCache = true;
+                PROGRAM.invalidGroupCache = true;
+            }
```

Comment:
nit let's inline these inside of Program.  Can save "public static void m() {}" (25 chars)

Reply:
sure. i just not sure where to put theis stuff. The beginning of main seems inapppropriate.

Reply:
i think the beginning of "RunThreads" might be reasonable

Reply:
sounds better than the beginning of main